.circleci/config.yml

```diff
@@ -43,6 +43,25 @@ jobs:
           root: .
           paths:
             - ./*
+  integration-test:
+    working_directory: ~/addons-frontend
+    machine: true
+    steps:
+      - checkout
+      - run:
+          name: Install Docker Compose
+          command: |
+            set -x
+            pip install docker-compose>=1.18
+            docker-compose --version
+      - run:
+          name: Install tox
+          command: pip install tox
+      - run:
+          name: Run tests
+          command: tox -e ui-test-ci
```

Comment:
As discussed on IRC, I think this makes more sense as:

```
command: ./tests/ui/run-functional-tests.sh
```

It needs a [shebang](https://stackoverflow.com/questions/10376206/what-is-the-preferred-bash-shebang).

I am suggesting the name `run-functional-tests.sh` because then anyone can run it locally, outside of circle-ci.